stdlib/3/subprocess.pyi

```diff
@@ -151,7 +151,29 @@ else:
                    start_new_session: bool = ...,
                    pass_fds: Any = ...) -> int: ...
 
-if sys.version_info >= (3, 4):
+if sys.version_info >= (3, 6):
+    # 3.6 added encoding
+    def check_output(args: _CMD,
+                     bufsize: int = ...,
+                     executable: _TXT = ...,
+                     stdin: _FILE = ...,
+                     stderr: _FILE = ...,
+                     preexec_fn: Callable[[], Any] = ...,
+                     close_fds: bool = ...,
+                     shell: bool = ...,
+                     encoding: str = ...,
```

Comment:
A couple of things:
- The type should be `Optional[str]` (None is a valid value)
- `errors` was added at the same time; could you add it too? (Another `Optional[str]`.)
- Contrary to what the documentation says, `encoding` and `errors` are keyword-only arguments, but among the other arguments only `timeout` and `input` are keyword-only (they are passed directly to Popen, which accepts positional arguments). Could you move `encoding` and `errors` to the end and add a `*` after `pass_fds`?
- Last, several of the other functions in this module also support the encoding and errors arguments. Could you add it there too?